src/components/LoginSignUp/LoginForm.js

```diff
@@ -81,6 +82,10 @@ const LoginForm = ( {
     navigation.getParent( )?.goBack( );
   };
 
+  const togglePasswordVisibility = () => {
+    setIsPasswordVisible( prevState => !prevState );
```

Comment:
This is peculiar. Why did you choose to write it this way and not `setIsPasswordVisible(  !isPasswordVisible )`?

Reply:
@jtklein Actually the toggle depeneds on the previous state so i thought it will be good to use in this way as it is considered a safer approach